pkg/controller/cronjob/utils.go

```diff
@@ -147,6 +150,65 @@ func getRecentUnmetScheduleTimes(cj batchv1beta1.CronJob, now time.Time) ([]time
 	return starts, nil
 }
 
+// getUnmetScheduleTimes gets the slice of all the missed times from the time a job
+// last was scheduled to up `now`.
+//
+// If there are too many (>100) unstarted times, it will raise a warning and but still return
+// the list of missed times.
+func getUnmetScheduleTimes(cj batchv1beta1.CronJob, now time.Time, schedule cron.Schedule, recorder record.EventRecorder) []time.Time {
```

Comment:
Do you even use the individual times?  If not, just return a count and determine that count with division, not a for loop.

Reply:
Is this method used anywhere other than unit tests?

Reply:
> Is this method used anywhere other than unit tests?

GitHub didn't load the diff.  I see it, but I'd still prefer you return only the count and the latest time to create for.  Keep your surface here small.